lib/utils.js

```diff
@@ -214,7 +214,7 @@ export function buildSQLQuery (base, payload = {}, operators = [], betweens = []
  *
  */
 export function isGeoPayloadOK (payload) {
-  const split = payload.coordinates.split(',');
+  const split = isArray(payload.coordinates) ? payload.coordinates : payload.coordinates.split(',');
```

Comment:
Ah, that's a 'relic' from the first implementation in which multiple values were comma separated. First thing we were doing was splitting comma separated values and storing them as an array.